akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1485,6 +1485,27 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
   def detach: javadsl.Flow[In, Out, Mat] = new Flow(delegate.detach)
 
   /**
+   * Materializes to `Future[Unit]` that completes on getting termination message.
+   * The Future completes with success when received complete message from upstream or cancel
+   * from downstream. It fails with the same error when received error message from
+   * downstream.
+   *
+   * Taking `Future` as materialization value for resulting `Flow`. In other words it's combine
+   * materialize values with Keep.right
```

Comment:
This extra paragraph confuses me, isn't what/how it materialize covered with the previous paragraph?